cpp/ql/src/experimental/Security/CWE/CWE-552/DangerousUseRename.cpp

```diff
@@ -0,0 +1,11 @@
+
+...
+  if (rename(from,to)==0) // BAD
+    return;
+...
```

Comment:
This example is now effectively empty. Is that what you wanted?

Reply:
yes I must have been too hasty